src/client/components/Error/Error.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+
+
+import Dialog from 'material-ui/Dialog';
+import FlatButton from 'material-ui/FlatButton';
+
+import './Error.scss';
+import actions from '../../actions';
+
+const { updateError } = actions;
+
+
+const Error = ({ error, dispatch }) => {
```

Comment:
as error is not required in propTypes, maybe pass it a default value like this :  
const Error = ({ .error = null, dispatch }) => {   ?  
It could prevent some trouble mabe, like the isErrorOpen value